org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/extensions/generators/ElementDeclaration.java

```diff
@@ -123,4 +125,12 @@ public void incrementOccurrences() {
 	public int getOccurrences() {
 		return occurrences;
 	}
+	
+	public boolean isDoneID() {

```

Comment:
I think that a Javadoc explaining what doneID is used for might be helpful.

Reply:
fixed, if you think it's bad, please suggest me an another javadoc